book/website/collaboration/team-manual.md

```diff
@@ -0,0 +1,203 @@
+They are in a better position to do so and have power over decision making. 
+This does not mean only senior members can be involved: the whole group should be able to provide input and it is important to hold discussions over points where there is no agreement in the group. 
+
+If you want to convince your supervisor that having a Team Manual is important: 
+- Check if a comparable lab is using a Lab Manual and show them this. 
+You can also use the list provided (see below [A list of Lab Manuals]) as an example. 
+- Team up with group members so that you're not the only person that wants to implement it
+- Start out with a simplified template or share a draft so that the supervisor can easily start
+
+There needs to be a balance between an extensive and detailed manual and an operable manual that can be implemented. 
+Consider the points listed in this overview as ideas to mix and match from in no particular order, and use and adapt those that make most sense in your particular setting.
+
+You can gather input on the Team Manual from group members via discussion sessions, or surveys. 
+See [this questionnaire](https://docs.google.com/forms/d/e/1FAIpQLSemv4CRIngTtXA39CH_ooZo17jMGQs_7RqyIkgLKmoJBLPlOA/viewform) for an example on how to engage lab members about their preferences. 
+ 
+(cl-team-manual-accountable)=
+## How do you keep each other accountable? 
+
+When a Team Manual is established it is important that everyone is on board.  
+If someone is not following the Lab Manual it needs to be discussed why this is the case. 
+It could be that the Lab Manual needs to be updated, or a discussion needs to take place to re-align the values of the group members, or perhaps this lab member is not a right fit for the group. 
+
+If someone breaks the agreements it needs to be clear what the consequences are. 
+The Team/Lab needs to be held accountable, as otherwise there is no point of having a Lab Manual. 
+See also [Patrick Lencioni's 'Teamwork: The Five Dysfunctions of a Team'](https://www.worldcat.org/title/794274430): 
+- Absence of trust
+- Fear of Conflict
+- Lack of commitment
+- Avoidance of accountability
+- Inattention to results
+
+(cl-team-manual-assess)=
+## How to assess your lab culture?
+* Leslie Vosshall's [lab survey](https://docs.google.com/forms/d/e/1FAIpQLScGCi7iACgmVBhFcE7G90oPwuTs-g9CQkrDmOUoQ4FvoT9CfA/viewform) to measure whether your lab is happy 
+* [Hernandez et al. 2021](https://doi.org/10.1101/2021.12.08.471870 )
+
+(cl-team-manual-examples)=
+## Examples of Team Manuals
+
+### Copy of a Team Manual template
+* [Team Manual Template](https://docs.google.com/document/d/1LqGdtHg0dMbj9lsCnC1QOoWzIsnSNRTSek6i3Kls2Ik/edit)
+
+### Examples of different types of Team/Lab Manuals
+* [Code of Conduct Whitaker lab](https://github.com/WhitakerLab/Onboarding/blob/master/CODE_OF_CONDUCT.md) and the [Basin Research Group](https://docs.google.com/document/d/10y7TP48ik1rcQBPA5Do8mZM7DJ5EbF0hyWP-csgb1QE/edit)
```

Comment:
The inclusive language warning is somewhat of a a "fluke" and can be ignored in my opinion - It flagged a Git ``master`` branch in a link (so in text that will not be rendered).